api/src/test/java/uk/gov/dhsc/htbhf/eligibility/service/v2/DWPClientV2Test.java

```diff
@@ -59,4 +65,76 @@ void shouldSendRequest() {
         verify(restTemplate).exchange(uri + DWP_ENDPOINT, HttpMethod.GET, httpEntity, IdentityAndEligibilityResponse.class);
         verify(getRequestBuilder).buildRequestWithHeaders(request);
     }
+
+    @Test
+    void shouldCreateSyntheticIdentifierForHouseholdWithChildren() {
+        //Given
+        DWPEligibilityRequestV2 request = DWPEligibilityRequestV2TestDataFactory.aValidDWPEligibilityRequestV2();
+        IdentityAndEligibilityResponse identityAndEligibilityResponse = identityAndEligibilityResponseWithHouseholdId("");
+        given(restTemplate.exchange(anyString(), any(), any(), eq(IdentityAndEligibilityResponse.class)))
+                .willReturn(new ResponseEntity<>(identityAndEligibilityResponse, OK));
+        HttpEntity httpEntity = aValidEligibilityHttpEntity();
+        given(getRequestBuilder.buildRequestWithHeaders(any())).willReturn(httpEntity);
+        //When
+        IdentityAndEligibilityResponse response = dwpClient.checkIdentityAndEligibility(request);
+        //Then
+        String syntheticIdComponents =
```

Comment:
Would this be clearer as the actual string of components rather than some code to build it?

Reply:
e.g. `String syntheticIdComponents = "BS14TQ2017-12-292015-06-23" or similar